flybywiresim/aircraft
# fix(atsu): added default callback on simbrief import

This fixes the simbrief status sticking in "READY" which effectively prevents a reinitialization when entering a new from/dest






Fixes #[issue_no]

## Summary of Changes

Assigned a dummy callback as default to prevent errors and added the `updateView` call removed in https://github.com/flybywiresim/a32nx/commit/fbfc98028414168dfaeffe91e6f439818888b390#diff-7594a1fb9a3e4167608b10794866786770e6aa1d8daf74ce35c213497f00d168L143 again

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Create a flightplan in simbrief
2. Go to the INIT page and import the flight plan
3. Enter a new from/to pair on the INIT page
4. The INIT REQ button should appear and you should be able to import a flightplan from simbrief again
5. Load passengers and verify the passenger count is according to the simbrief OFP.
6. Repeat the above with 2-3 different Simbrief flight plans


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
